app/models/concerns/pre_sale_prices/line_item_pre_saleable.rb

```diff
@@ -19,7 +19,7 @@ def copy_pre_sale_price
     def update_pre_sale_price
       variant_pre_sale_price = variant.pre_sale_price_in(variant.currency).amount
 
-      if variant_pre_sale_price > 0
+      if variant_pre_sale_price.present? and variant_pre_sale_price > 0
```

Comment:
Yeah I thought about setting a default. Seems there are some cases where we want it to be nil though. @chrisblackburn can probably explain better than me.